### ISSUE TITLE
SNAP Network -> GraphSAGE JSON files

### ISSUE THREAD

**Issue by sidmasta11** [williamleif/GraphSAGE#96]
Hello,

What is the best way to create the various input files necessary from a SNAP Network with attributes for unsupervised learning and supervised learning?

Is the best practice to go from SNAP Network -> NetworkX Object -> GraphSAGE JSON files or SNAP Network -> GraphSAGE JSON files?

Thank You

**Comment by RexYing** [comment]
The GraphSAGE json files are just numpy arrays of features, adjacency etc. So you don't have to go through the intermediate networkx format. Simple read the text file of SNAP network, and convert it into numpy arrays, before using json.dump

**Comment by cj2001** [comment]
I agree with the original post that this section (and the answer given above) is not very clear in the `README.md`.  Can you please provide more detail on this format?

**Comment by RexYing** [comment]
Is there anything other than the input format section in README needed? what details and which format you are not sure about?
